xamarin/Xamarin.Forms
# Remove StateTriggersExperimental flag

### Description of Change ###

Remove StateTriggersExperimental flag!

### API Changes ###

Forms.SetFlags ("StateTriggers_Experimental") is no longer required; to use `StateTriggers`.

### Platforms Affected ### 

- Core/XAML (all platforms)

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the StateTriggers Gallery. Without set the flag everything should works without problems.

### PR Checklist ###
